TinYard/Extensions/CallbackTimer/Impl/VO/Timer.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace TinYard.Extensions.CallbackTimer.Impl.VO
+{
+    public class Timer
+    {
+        public Action TimerCallback { get { return _timerCallback; } }
```

Comment:
I have talk to people be who didn't like this method of getter and setters (having a private variable instead or private set). IMO it's fine. Just something to keep in mind.

Reply:
Good spot. I usually switch between them but should be consistent here and do like I prefer with a private setter.